bpf/cluster_config.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
+/* Copyright Authors of Cilium */
+
+#ifndef __CLUSTER_CONFIG__
+#define __CLUSTER_CONFIG__
+
+/*
+ *
+ *
+ *                     **** WARNING ****
+ * This is just a dummy header with dummy values to allow for test
+ * compilation without the full code generation engine backend.
+ *
+ *
+ *
+ */
+#define CLUSTER_ID_LEN 8
+#define CLUSTER_ID_MAX 255
+#define IDENTITY_LEN 16
+#define IDENTITY_MAX 65535
```

Comment:
we don't need to write them to the config, but I thought its usage is a little more concise (using `CLUSTER_ID_MAX` rather than `CLUSTER_ID_MAX(CLUSTER_ID_LEN)`.
What do you think about defining the MAX macros in common.h instead, and only writing the LEN defines to the header?

Reply:
actually, it may not make sense to do this in the way i described. the maximum cluster ID is what is actually passed in via flag. The len of each is calculated once using the Getters added in [this commit](https://github.com/cilium/cilium/pull/27520/commits/ac6778cb54812db8ce35c84fae03bf541f88f18e). I think it's not explicitly necessary to define all 4 in the header, but it seems a bit simpler

Reply:
I don't have a strong opinion, it just seemed a bit redundant to specify both `X_LEN` and `X_MAX` as part of the config. But if that is more convenient, then it is fine for me.